data/korath/korath outfits.txt

```diff
@@ -652,7 +652,141 @@ outfit "Steering (Stellar Class)"
+
+outfit "Arkrof GP Hybrid Steering"
+	plural "Arkrof GP Hybrid Steerings"
+	category "Engines"
+	"cost" 131000
+	thumbnail "outfit/tiny efreti steering"
+	"mass" 12
+	"outfit space" -12
+	"engine capacity" -12
+	"turn" 323
+	"turning energy" 1.2
+	"turning heat" 1.225
+	"steering flare sprite" "effect/efreti flare/tiny"
+		"frame rate" 8
+	"steering flare sound" "ion tiny"
+	description "Unlike most Korath engines, Efreti engines do not rely solely on plasma. Gravitoplasma engines combine traditional Korath plasma thrust with Quarg-inspired forays into gravitation-based propulsion. The smallest steering manages to develop a great deal of torque at the expense of higher energy costs."
+
+outfit "Farves GP Hybrid Steering"
+	plural "Farves GP Hybrid Steerings"
+	category "Engines"
+	"cost" 413000
+	thumbnail "outfit/small efreti steering"
+	"mass" 24
+	"outfit space" -24
+	"engine capacity" -24
+	"turn" 737
+	"turning energy" 2.5
+	"turning heat" 2.75
+	"steering flare sprite" "effect/efreti flare/small"
+		"frame rate" 7.5
+	"steering flare sound" "ion small"
+	description "Exposure to the Quarg's graviton-thrusters, coupled with the forced retreat from research into interstellar jump drive technology, has left the Efreti engineers with little else to tinker with than subluminal propulsion. Their ongoing efforts represent their evolving understanding of fundamental physics and how it can be put into practice."
+	description "	These resulting steering thrusters are one important product of this effort. This compact steering system is a classic example of Korath technology: small, powerful, and woefully inefficient. It appears that the Quarg influence on the Efreti has not changed the basic design philosophy of the Korath living here."
+
+outfit "Gaktem GP Hybrid Steering"
+	plural "Gaktem GP Hybrid Steerings"
+	category "Engines"
+	"cost" 833000
+	thumbnail "outfit/medium efreti steering"
+	"mass" 42
+	"outfit space" -42
+	"engine capacity" -42
+	"turn" 1400
+	"turning energy" 4.325
+	"turning heat" 5.15
+	"steering flare sprite" "effect/efreti flare/medium"
+		"frame rate" 7
+	"steering flare sound" "ion medium"
+	description "While Korath understanding of the ultimate origins of spacetime are still incomplete, their grudging divorce from jump drive technology has pushed Efreti scientists and engineers to begin to instantiate their unified understandings of quantum gravity and the nuclear forces within their newer hybrid gravitoplasma thrusters."
+	description "	These Efreti steering thrusters can fit into just about any ship. Yet, unless accompanied by some Korath power systems as well, their energy consumption will often be more than the ship can handle. Their main advantage when compared to other Korath engines of the same size is their reduced heat production."
```

Comment:
Having a mass of 42 doesn't suggest to me that these steering thrusters can `fit into just about any ship.` That's fairly large and outside of the range for many ships. Perhaps it would be better to put this in the Arkrof steering description instead?

Reply:
These descriptions are quite lengthy, I'll just cut that bit.